client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/shipping-rate.js

```diff
@@ -45,7 +45,12 @@ class ShippingRate extends Component {
 			}
 		}
 		if ( includedServices.insurance ) {
-			servicesToRender.push( translate( 'Insurance (up to %s)', { args: [ formatCurrency( includedServices.insurance, 'USD') ] } ) );
+			const numericInsurance = Number( includedServices.insurance );
```

Comment:
This seems safe, because the service is never going to return `insurance` as `undefined` 👍   (commenting because `Number( undefined) // NaN`)

Reply:
Good call. `insurance` is not required so it could be `undefined`, but it won't make it pass line 47's `if ( includedServices.insurance )` check since it will be `falsy`.

Reply:
I missed that, even better!